### DISCUSSION TITLE
删除工作资料的功能来的太迟了

### DISCUSSION THREAD

**Discussion by starrynight168** [BinTianqi/OwnDroid#58] [Show and tell]
两天前我想双开个软件，但是接管了设备所有者权限就不可以双开，然后我就取消了设备所有者。然后就新出了个工作资料的窗口，我就创建了工作资料，但默认由组织接管的。但是问题来了，我想取消发现删除不掉工作资料了，系统设置那里的删除工作资料显示被it管理员禁用。然后我就在软件里找，但是根本就没有删除工作资料的入口。教程文档也没有写怎么删除工作资料。（此时我还不知道用adb可以删除工作资料）后来我看就有个清楚数据，当时好像是写的是此操作在开启工作资料里使用可以删除工作资料。我以为只是会删除工作资料的所有数据，没想到是删除设备的所有数据。所以就完蛋了……
如今删除工作资料终于来了，可惜来迟了😭😭😭 提个建议，在清楚数据那里加些提示，和二次确认的这样子尤其是静默删除。![Screenshot_2024-07-19-11-08-02-27_df198e732186825c8df26e3c5a10d7cd.png](https://github.com/user-attachments/assets/65dca8e2-3401-4a72-97bd-db59102fddbe)

![Screenshot_2024-07-19-11-09-17-27_df198e732186825c8df26e3c5a10d7cd.png](https://github.com/user-attachments/assets/4d9190da-5ed7-473d-a38b-472d66982cdb)

**Comment by BinTianqi** [comment]
你现在看的文档是5.6的，以前的文档写了，工作资料里面清除数据就是删除工作资料。

#53 有人建议默认隐藏清除数据，5.6之后想清除数据，需要在设置中打开显示危险功能
